bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -1036,6 +1038,11 @@ mod test {
         let settings3 = machine.get_room_settings("!XYZ:matrix.org".into())?;
         assert!(settings3.is_none());
 
+        assert!(machine.is_user_tracked("@ganfra146:matrix.org".into()).unwrap());
+        assert!(machine.is_user_tracked("@Amandine:matrix.org".into()).unwrap());
+        assert!(machine.is_user_tracked("@this-is-me:matrix.org".into()).unwrap());
+        assert!(machine.is_user_tracked("@ganfra:matrix.org".into()).unwrap());
```

Comment:
Can you also add an assert for what's expected for `NotAUser%ID`, if that makes sense?

Reply:
I can't exactly as is_user_track will bubble the user parse error.